### PR TITLE
fix(cron): add check objectives task

### PR DESCRIPTION
**Pull request by VladislavSokov**

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -4,7 +4,11 @@
   "cron": [
     {
       "command": "rake top_up_accounts",
-      "schedule": "0 14 * * *"
+      "schedule": "0 14 * * 5"
+    },
+    {
+      "command": "rake check_objectives",
+      "schedule": "0 9-21/2 * * *"
```
Comment (ka8725) [inline app.json:11]: why only 9-12? why around the clock? I think every hour is ok. or every 30 min.

Comment (ka8725) [reply app.json:11]: We also could run it on demand in the points of the app that would affect it - on transaction added. And only for the affected account.

Comment (VladislavSokov) [reply app.json:11]: Good idea, I'll think about it.

**Review by ka8725 — APPROVED** [review]
Looks awesome! Thanks!

**Pull request merged**